typescript/infra/src/config/environment.ts

```diff
@@ -44,6 +45,7 @@ export type EnvironmentConfig = {
   ) => Promise<MultiProvider>;
   helloWorld?: Partial<Record<Contexts, HelloWorldConfig>>;
   keyFunderConfig?: KeyFunderConfig;
+  liquidityLayerConfig?: ChainMap<BridgeAdapterConfig>;
```

Comment:
Consider merging the liquidityRelayerConfig and the liquidityLayerConfig, so that we can enforce that if there is a liquidityLayerConfig there *must* be a liquidityRelayer config.

e.g.

```
export type LiquidityLayerEnvConfig = {
  relayer: LiquidityLayerRelayerConfig,
  bridgeAdapters: ChainMap<BridgeAdapterConfig>
}

export type EnvironmentConfig = {
...
  liquidityLayer?: LiquidityLayerEnvConfig
}
```

Reply:
Great suggestion